sentry/values.yaml

```diff
@@ -108,6 +108,10 @@ sentry:
     # tolerations: []
     # podLabels: []
     # commitBatchSize: 1
+  cleanup:
+    enabled: false
```

Comment:
Thanks for your PR.

I think we should enable this per default ?

Reply:
Done!